FAQsCategories.ascx.cs

```diff
@@ -158,85 +155,27 @@ protected void cmdCancel_Click(System.Object sender, System.EventArgs e)
             panelAddEdit.Visible = false;
         }
 
-        /// <summary>
-        /// Handles the NodeDataBound event of the treeCategories control (adds Tooltip)
-        /// </summary>
-        /// <param name="sender">The source of the event.</param>
-        /// <param name="e">instance containing the event data.</param>
-        protected void treeCategories_NodeDataBound(object sender, RadTreeNodeEventArgs e)
-        {
-            e.Node.ToolTip = (string)DataBinder.Eval(e.Node.DataItem, "FaqCategoryDescription");
-            e.Node.Value = DataBinder.Eval(e.Node.DataItem, "FaqCategoryId").ToString();
-            e.Node.ImageUrl = IconController.IconURL("folder");
-        }
-
-        /// <summary>
-        /// Handles the NodeClick event of the treeCategories control
-        /// </summary>
-        /// <param name="sender">The source of the event.</param>
-        /// <param name="e">instance containing the event data.</param>
-        protected void treeCategories_NodeClick(object sender, RadTreeNodeEventArgs e)
-        {
-            int faqCategoryId = Convert.ToInt32(e.Node.Value);
-            EditCategory(faqCategoryId);
-            cmdDelete.Visible = true;
-        }
+        #endregion
 
+        #region Private Methods
 
-        protected void treeCategories_HandleDrop(object sender, RadTreeNodeDragDropEventArgs e)
```

Comment:
If I understand correctly we loose the ability to order categories by removing this right ?

Reply:
Yes, it remove the ability for Drag and drop, because asp.net treeview doesn't support it.